modules/gapi/samples/one_vpl_source_to_bgr_conv.cpp

```diff
@@ -0,0 +1,68 @@
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <cctype>
+#include <tuple>
+#include <memory>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/gpu/ggpukernel.hpp>
+#include <opencv2/gapi/streaming/onevpl/source.hpp>
+#include <opencv2/gapi/streaming/onevpl/data_provider_interface.hpp>
+#include <opencv2/gapi/streaming/onevpl/default.hpp> // default VPL source
+#include <opencv2/highgui.hpp> // CommandLineParser
+#include <opencv2/gapi/ocl/core.hpp>
+
+const std::string about =
+    "This is an example presents decoding on GPU using VPL Source and passing it to OpenCL backend";
+const std::string keys =
+    "{ h help   |                           | Print this help message }"
+    "{ input    |                           | Path to the input video file. Use .avi extension }"
+    "{ output   | output.avi                | Path to the output RAW video file. Use .avi extension }";
+
+int main(int argc, char *argv[]) {
+    cv::CommandLineParser cmd(argc, argv, keys);
+    cmd.about(about);
+    if (cmd.has("help")) {
+        cmd.printMessage();
+        return 0;
+    }
+
+    // Get file name
+    const auto input = cmd.get<std::string>("input");
+    const auto output = cmd.get<std::string>("output");
+
+    // Make a OneVPL capture with default configuration
+    cv::gapi::wip::onevpl::DefaultVPLSource source(input,
```

Comment:
@smirnov-alexey , @TolyaTalamanov 
Why do we need another source ?
Once again - I see no necessity in implementation another one entity like `DefaultVPLSource ` and redundant auxiliary types like `VPLSourcePlatform`

Please use true-default source with `make_vpl_source`
Example
```
auto cap = make_vpl_source(input, { CfgParams::create_aceleration_type("MFX_ACCEL _TYPE_D3D11")});
```

It seems not longer than your version, but the more healthful

Reply:
I'm renaming it to something else to avoid the confusion. We need a source plus 2 extra pointers connected to that source for remote context. Unfortunately, we can't just go with `make_vpl_source(input, { CfgParams::create_aceleration_type("MFX_ACCEL _TYPE_D3D11")});`